web/src/HUD.tsx

```diff
@@ -332,6 +332,18 @@ export function HUDFromContext(props: React.PropsWithChildren<{}>) {
   return <HUD history={history} interfaceVersion={interfaceVersion} />
 }
 
+function compareObjects(a: any, b: any): number {
```

Comment:
(optional nitpick) What about a more descriptive name, like `compareObjectsOrder`, so it's easier to tell how this function is being used?

It may not be worth the Typescript finagling, but I _think_ it would work to type this as `<T extends { status?: Proto.v1alpha1UIResourceStatus, metadata?: Proto.v1ObjectMeta }>`? 🤞🏻

Reply:
i like that, done!

got something that kinda works...